main.go

```diff
@@ -315,14 +295,25 @@ func main() {
 			default:
 			}
 
-			opts := link.KprobeMultiOptions{Symbols: funcsByPos[pos]}
+			addrs := make([]uintptr, 0, len(fns))
+			for _, fn := range fns {
+				if addr, ok := addr2name.Name2AddrMap[fn]; ok {
```

Comment:
As this is no longer 1-1 mapping, maybe we could change `Name2AddrMap` to store `map[string][]uintptr`?

Reply:
Thanks for pointing it out. I confirm it on 6.5.0-rc4+ kernel:

```bash
# ./pwru --output-meta --output-tuple icmp
2023/12/20 16:23:42 Attaching kprobes (via kprobe-multi)...
2023/12/20 16:23:42 Attaching kprobe-multi for pos 3 (116 functions / 116 addrs)
2023/12/20 16:23:42 Attaching kprobe-multi for pos 1 (953 functions / 956 addrs)
2023/12/20 16:23:42 Attaching kprobe-multi for pos 4 (12 functions / 12 addrs)
2023/12/20 16:23:42 Attaching kprobe-multi for pos 2 (350 functions / 350 addrs)
2023/12/20 16:23:42 Attaching kprobe-multi for pos 5 (31 functions / 31 addrs)
```